packages/shared/src/components/widgets/WidgetCard.tsx

```diff
@@ -1,19 +1,20 @@
-import React, { PropsWithChildren, ReactElement } from 'react';
+import React, { ComponentProps, ReactElement } from 'react';
 import classNames from 'classnames';
 
-interface WidgetCardProps extends PropsWithChildren {
+interface WidgetCardProps extends ComponentProps<'section'> {
   heading?: string;
   className?: string;
 }
 
 export const WidgetCard = (props: WidgetCardProps): ReactElement => {
-  const { heading, className, children } = props;
+  const { heading, className, children, ...attrs } = props;
   return (
     <section
       className={classNames(
         'flex flex-col rounded-16 border border-border-subtlest-tertiary',
         className,
       )}
+      {...attrs}
```

Comment:
What is this for?

Reply:
I'm using it to pass potential attributes that comes from `ComponentProps<'section'>`. 

I'm passing  `data-testid="related-tags"` in Related tags for eg.